src/yunohost/dyndns.py

```diff
@@ -252,6 +226,30 @@ def dyndns_update(dyn_host="dyndns.yunohost.org", domain=None, key=None,
         'zone %s' % host,
     ]
 
+
+    old_ipv4 = check_output("dig @91.224.148.92 +short %s" % domain).strip()
+    old_ipv6 = check_output("dig @91.224.148.92 +short aaaa %s" % domain).strip()
```

Comment:
It looks like every YunoHost is going to make a request to a DNS server every 2min, isn't that a lot for those DNS server? I could be like ~3000 requests every 2min (if everyone has updated), I'm not sure it's good.

Reply:
After an IRC discussion that ~should~ be alright, we are hitting our own DNS servers.

Reply:
I'm still wondering if we shouldn't switch a DNS resolution here, this IP is own by tetaneutral and if we lose it for wathever reason all the dyndns domains are going to be in a shitty situation while if we bind that to the domain we own, it would be way more safe.

Reply:
+1